### PR TITLE
Fix static framework packaging

### LINKED ISSUE #5071: Not possible to connect a static framework >= 3.3 alpha-1
**Platform:** iOS
**Mapbox SDK version:** >= 3.3 alpha-1

Starting with version 3.3 alpha-1 is not possible to connect a static framework. Perhaps additional settings are needed? Version 3.2 connects without problems.

### Steps to trigger behavior

 1. run make ipackage-strip
 2. get the static framework
 3. link with an app

### Log

```
Ld /Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/Taxovichkof.app/Taxovichkof normal armv7
    cd "/Users/MacBook/Google Диск/iOS/TaxiApp"
    export IPHONEOS_DEPLOYMENT_TARGET=7.0
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch armv7 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS9.3.sdk -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/AFNetworking -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/Bolts -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/Facebook-iOS-SDK -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/Google-API-Client -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/OpenInChrome -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/PKMultipartInputStream -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/SAMTextView -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/SDWebImage -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/SSKeychain -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/ShareKit -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/UIActivityIndicator-for-SDWebImage -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/gtm-http-fetcher -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/gtm-logger -L/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/gtm-oauth2 -L/Users/MacBook/Google\ Диск/iOS/TaxiApp/Pods/GoogleAnalytics/Libraries -L/Users/MacBook/Google\ Диск/iOS/TaxiApp/Pods/GoogleConversionTracking/Libraries -L/Users/MacBook/Google\ Диск/iOS/TaxiApp/Pods/GoogleIDFASupport/Libraries -L/Users/MacBook/Google\ Диск/iOS/TaxiApp/Libraries/metrica-sdk-ios-master -L/Users/MacBook/Google\ Диск/iOS/TaxiApp/Libraries/ShareKit-master/Submodules/buffer-ios-sdk/BufferSDK -F/Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos -F/Users/MacBook/Google\ Диск/iOS/TaxiApp/Pods/GoogleMaps/Frameworks -F/Users/MacBook/Google\ Диск/iOS/TaxiApp/Pods/ShareKit/Frameworks -F/Users/MacBook/Google\ Диск/iOS/TaxiApp/Pods/YandexMobileMetrica -F/Users/MacBook/Google\ Диск/iOS/TaxiApp -F/Users/MacBook/Google\ Диск/iOS/TaxiApp/Libraries -F/Users/MacBook/Google\ Диск/iOS/TaxiApp/Libraries/mapbox-ios-sdk-3/static -F/Users/MacBook/Google\ Диск/iOS/TaxiApp/Libraries/ShareKit-master/Frameworks -F/Users/MacBook/Google\ Диск/iOS/TaxiApp/Libraries/ShareKit-master/Frameworks/Pinterest.embeddedframework -F/Users/MacBook/Google\ Диск/iOS/TaxiApp/Libraries/QuincyKit-develop/client/iOS -F/Users/MacBook/Google\ Диск/iOS/TaxiApp/Libraries/FacebookSDKs-iOS-20150910 -filelist /Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Intermediates/Taxi.build/Debug-iphoneos/Taxovichkof.build/Objects-normal/armv7/Taxovichkof.LinkFileList -Xlinker -rpath -Xlinker @executable_path/Frameworks -miphoneos-version-min=7.0 -dead_strip -Xlinker -no_deduplicate -lxml2 -ObjC -lAFNetworking -lAdIdAccessLibrary -lBolts -lFacebook-iOS-SDK -lGoogle-API-Client -lGoogleAnalytics -lGoogleConversionTracking -lOpenInChrome -lPKMultipartInputStream -lSAMTextView -lSDWebImage -lSSKeychain -lShareKit -lUIActivityIndicator-for-SDWebImage -lc++ -lgtm-http-fetcher -lgtm-logger -lgtm-oauth2 -licucore -lsqlite3 -lz -framework AVFoundation -framework Accelerate -framework AdSupport -framework AddressBook -framework AssetsLibrary -framework CoreBluetooth -framework CoreData -framework CoreGraphics -framework CoreLocation -framework CoreMedia -framework CoreMotion -framework CoreTelephony -framework CoreText -framework Foundation -framework GLKit -framework GoogleMaps -framework GooglePlus -framework ImageIO -framework MediaPlayer -framework MessageUI -framework MobileCoreServices -framework OpenGLES -framework QuartzCore -framework Security -framework Social -framework SystemConfiguration -framework Twitter -framework UIKit -framework YandexMobileMetrica -weak_framework Accounts -weak_framework AudioToolbox -weak_framework CoreGraphics -weak_framework CoreLocation -weak_framework Foundation -weak_framework QuartzCore -weak_framework SafariServices -weak_framework Security -weak_framework Social -weak_framework UIKit -fobjc-arc -fobjc-link-runtime -lz -lsqlite3 -lc++ -framework QuartzCore -framework ImageIO -framework GLKit -framework AdSupport -framework AudioToolbox -framework CrashReporter -framework MobileCoreServices -framework Security -framework Mapbox -framework SystemConfiguration -framework CoreMotion -framework UIKit -framework MessageUI -framework SafariServices -framework CoreMedia -framework CoreFoundation -framework Foundation -framework AVFoundation -framework AssetsLibrary -framework Accounts -framework MediaPlayer -framework Social -framework CoreData -framework OpenGLES -framework CoreText -framework MapKit -framework LiveSDK -lPods-Taxovichkof -Xlinker -dependency_info -Xlinker /Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Intermediates/Taxi.build/Debug-iphoneos/Taxovichkof.build/Objects-normal/armv7/Taxovichkof_dependency_info.dat -o /Users/MacBook/Library/Developer/Xcode/DerivedData/Taxi-erbmhgfrlvababhfavyiqjkewgeq/Build/Products/Debug-iphoneos/Taxovichkof.app/Taxovichkof

Undefined symbols for architecture armv7:
  "_OBJC_METACLASS_$_MGLPointAnnotation", referenced from:
      _OBJC_METACLASS_$_TAXMapLocation in TAXMapLocation.o
  "_OBJC_CLASS_$_MGLPointAnnotation", referenced from:
      _OBJC_CLASS_$_TAXMapLocation in TAXMapLocation.o
  "_OBJC_CLASS_$_MGLAnnotationImage", referenced from:
      objc-class-ref in TAXOrderViewController.o
      objc-class-ref in TAXCarsAnimation.o
      objc-class-ref in TAXRadarViewController.o
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

### PR DESCRIPTION
**Pull request by friedbunny**
Fixes #5071, where `make ipackage` would produce a static framework that did not include our Objective-C symbols.

- Adds `libMapbox.a` to the libtool binary conglomeration step for the static framework.
- Simulator debug builds now include `x86_64` slices (by setting `ONLY_ACTIVE_ARCH=NO`).
- Per 9732f02422eb5cd5852d47e888b061aaf6ffe0bd (#5120), the default build type is now debug for `make ipackage-foo`.
 - The deploy script has been updated to explicitly build for release.
 - For manual release builds, run `make ipackage-foo BUILDTYPE=Release`.

/cc @1ec5 @boundsj

### CODE REVIEW

**Comment by 1ec5** [comment]
👍

**Comment by friedbunny** [comment]
@1ec5 reminded me that I should mention a weird issue I encountered while trying to use libtool to smoosh together the constituent libraries: When the simulator libraries did not include an `x86_64` slice (because `BUILDTYPE=Debug` defaults to `ONLY_ACTIVE_ARCH=YES`), the resulting "fat" file from libtool would only include the symbols from libgeojsonvt.a.

I don’t quite understand why, but libtool became able to combine all of the constituent libraries once I set `ONLY_ACTIVE_ARCH=NO`.

**Comment by friedbunny** [comment]
Another note, with a hat-tip to @1ec5: `otool -L lib.a` and `lipo -info lib.a` are very useful for identifying what classes/sub-libraries and architectures a library has (respectively).

```
$ otool -L build/ios/pkg/static/Mapbox.framework/Mapbox
Archive : build/ios/pkg/static/Mapbox.framework/Mapbox
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLPointAnnotation.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLFeature.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(NSBundle+MGLAdditions.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLAPIClient.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLOfflinePack.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLUserLocation.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(SMCalloutView.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLClockDirectionFormatter.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLShapeCollection.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLShape.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(NSString+MGLAdditions.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLStyle.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLGeometry.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLMultiPoint.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLCompassDirectionFormatter.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(NSValue+MGLAdditions.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLTypes.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLCoordinateFormatter.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLAnnotationImage.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLLocationManager.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLMapView.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLMapCamera.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLAnnotationView.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLUserLocationAnnotationView.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(reachability.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLMapboxEvents.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLPolygon.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(NSProcessInfo+MGLAdditions.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLOfflineStorage.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLCompactCalloutView.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLTilePyramidOfflineRegion.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLAccountManager.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(MGLPolyline.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(clipper.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(csscolorparser.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(generate_clip_ids.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(annotation_manager.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(annotation_tile.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(point_annotation_impl.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(shape_annotation_impl.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(circle_buffer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(collision_box_buffer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(debug_font_buffer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(elements_buffer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(feature_index.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(fill_buffer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(glyph_atlas.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(icon_buffer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(line_atlas.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(line_buffer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(static_vertex_buffer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(text_buffer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(vao.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(debugging.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(gl.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(gl_config.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(object_store.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(texture_pool.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(background_layer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(circle_layer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(custom_layer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(fill_layer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(line_layer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(raster_layer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(symbol_layer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(camera.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(map.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(transform.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(transform_state.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(view.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(log.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(circle_bucket.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(debug_bucket.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(fill_bucket.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(frame_history.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(line_bucket.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(painter.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(painter_background.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(painter_circle.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(painter_clipping.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(painter_debug.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(painter_fill.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(painter_line.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(painter_raster.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(painter_symbol.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(raster_bucket.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(symbol_bucket.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(circle_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(collision_box_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(icon_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(line_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(linepattern_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(linesdf_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(outline_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(outlinepattern_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(pattern_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(plain_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(raster_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(sdf_shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(shader.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(uniform.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(source.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(sprite_atlas.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(sprite_image.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(sprite_parser.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(sprite_store.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(network_status.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(resource.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(response.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(class_dictionary.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(function_evaluator.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(property_parsing.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(style.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(style_bucket_parameters.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(style_layer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(style_parser.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(types.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(check_max_angle.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(collision_feature.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(collision_tile.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(get_anchors.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(glyph.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(glyph_pbf.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(glyph_set.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(glyph_store.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(quads.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(shaping.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(geojson_tile.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(geometry_tile.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(raster_tile_data.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(tile.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(tile_cache.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(tile_data.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(tile_id_io.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(tile_worker.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(vector_tile.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(vector_tile_data.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(chrono.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(clip_id.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(clip_lines.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(compression.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(constants.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(default_styles.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(dtoa.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(geo.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(grid_index.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(http_header.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(intersection_tests.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(io.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(mapbox.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(mat2.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(mat3.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(mat4.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(math.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(merge_lines.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(premultiply.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(raster.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(stopwatch.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(thread_context.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(tile_coordinate.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(tile_cover.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(transition.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(url.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(version_info.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(work_queue.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(work_request.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(work_task.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(worker.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(parsedate.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(headless_display.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(headless_view.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(headless_view_eagl.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(async_task.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(run_loop.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(timer.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(asset_file_source.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(default_file_source.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(online_file_source.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(offline.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(offline_database.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(offline_download.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(sqlite3.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(http_file_source.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(log_nslog.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(string_nsstring.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(image.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(nsthread.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(reachability.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(geojsonvt.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(clip.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(convert.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(simplify.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(transform.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(tile.o):
build/ios/pkg/static/Mapbox.framework/Mapbox(wrap.o):
```

```
$ lipo -info build/ios/pkg/static/Mapbox.framework/Mapbox
Architectures in the fat file: build/ios/pkg/static/Mapbox.framework/Mapbox are: armv7 arm64 i386 x86_64 armv7s 
```

File: platform/ios/scripts/package.sh [hunk 1]
```diff
@@ -84,6 +84,7 @@ xcodebuild \
     CURRENT_SHORT_VERSION=${SHORT_VERSION} \
     CURRENT_SEMANTIC_VERSION=${SEM_VERSION} \
     CURRENT_COMMIT_HASH=${HASH} \
+    ONLY_ACTIVE_ARCH=NO \
```
Comment (1ec5) [inline platform/ios/scripts/package.sh]: We should revisit whether this flag is necessary after mapbox/mason#181 lands. Ideally we’d stick to just the standard architectures, to minimize the build size.

**Pull request merged**